documentation/project-docs/developer-guide.md

```diff
@@ -33,6 +33,12 @@ As part of the build, some intermediate files will get generated which may run i
 
 To open the solution in Visual Studio, be sure to build with `build.cmd` and run the generated environment for your shell. If you're using `cmd`, then run `artifacts\sdk-build-env.bat`. If you're using powershell, you need to 'dot source' `artifacts/sdk-build-env.ps1`. Finally, open Visual Studio with `devenv sdk.sln`.
 
+In addition, Visual Studio must have the following option set:
```

Comment:
please add a note in the testing section as well. You probably don't need the picture in both places.

Reply:
Done and done 🫡